tests/main.cpp

```diff
@@ -58,9 +59,13 @@ TEST_CASE("Simulate & Send Transaction") {
 TEST_CASE("Request Airdrop") {
   const solana::Keypair keyPair = solana::Keypair::fromFile(KEY_PAIR_FILE);
   auto connection = solana::rpc::Connection(DEVNET);
+
+  auto prev_sol = connection.getBalance(keyPair.publicKey);
   auto result = connection.requestAirdrop(keyPair.publicKey, 1000000000);
-  // TODO: validate using confirmTransaction 
-  CHECK_GT(result.length(), 0);
+  std::this_thread::sleep_for(std::chrono::seconds(15));
```

Comment:
please use confirmTransaction for this

Reply:
oh i see in the next issue, that works with me

Reply:
yum will update this test after the confirmTransaction has been implmented properly